README.md

```diff
@@ -220,6 +222,9 @@ Initializes the server views manager where:
               where `callback` has the signature `function(err, compiled)` where `compiled` is a
               function with signature `function(context, options, callback)` and `callback` has the
               signature `function(err, rendered)`.
+            - `prepare(config)` - initializes additional engine state.
```

Comment:
I see no reason that it needs to be -- I just didn't have an async example on hand. I'm happy to rework it to be optionally async (and admittedly more consistent with the rest of the API). This seems to suggest a `prepareMode` option if it is going to parallel the compile function. Do you have any thoughts on this approach?